test/flex-justify-content.tsx

```diff
@@ -18,7 +17,7 @@ test('row - align text to center', t => {
 test('row - align multiple text nodes to center', t => {
 	const output = renderToString(
 		<Box justifyContent="center" width={10}>
-			A{'B'}
+			AB
```

Comment:
`AB` is not the same as `A{'B'}`. `A{'B'}` is rendered as two different text nodes and then combined. Why is this test changing?

Reply:
prettier got a little too helpful here. I'll revert it back.

Reply:
It was actually `xo --fix` that formatted it. I restored it back as well as another instances in the same file. https://github.com/vadimdemedes/ink/pull/264/commits/eaf6c6d8ae156def8a1351276022c64f295cfc29

Reply:
Can you open an issue on https://github.com/yannickcr/eslint-plugin-react if there isn't already one?

Reply:
Based on https://github.com/yannickcr/eslint-plugin-react/blob/7dce90a4f570847b33eaefddb8d9bb7d6eca9dc5/docs/rules/jsx-curly-brace-presence.md, `xo --fix`correcting `A{'B'}`to `AB`is correct behavior. 

In our case, we're intentionally causing this scenario. To prevent fix from correcting it, we should either disable globally as I've done or locally on each line/file. What would you prefer?